pkg/resources/management.cattle.io/v3/setting/validator.go

```diff
@@ -194,6 +197,53 @@ func (a *admitter) validateAuthUserSessionTTLMinutes(s *v3.Setting) error {
 	return nil
 }
 
+// validateAuthUserSessionIdleTTLMinutes validates the auth-user-session-idle-ttl-minutes setting
+// to make sure it's a positive integer and that duration is not greater than
+// auth-user-session-ttl-minutes settings if they are set.
+// If it encounters an error fetching or parsing auth-user-session-ttl-minutes settings
+// it logs but doesn't return the error to avoid rejecting the request.
+func (a *admitter) validateAuthUserSessionIdleTTLMinutes(s *v3.Setting) error {
+	if s.Value == "" {
+		return nil
+	}
+
+	userSessionIdleDuration, err := parseMinutes(s.Value)
+	if err != nil {
+		return field.TypeInvalid(valuePath, s.Value, err.Error())
+	}
+	if userSessionIdleDuration < 1 {
+		return field.TypeInvalid(valuePath, s.Value, "negative value or less than 1 minute")
+	}
+
+	isGreaterThanSetting := func(name string) bool {
+		setting, err := a.settingCache.Get(name)
+		if err != nil {
+			logrus.Warnf("[settingValidator] Failed to get %s: %s", name, err)
+			return false // Deliberately allow to proceed.
```

Comment:
Is there any chance of allowing the setting to be greater than auth-user-session-ttl-minutes by ignoring the error here?  Or is it "not all that bad" and we accept the risk?